packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -84,10 +92,14 @@ export const messages = {
   parameter_gatherer_placeholder_delete_default_org:
     'Confirm to continue deleting the default org',
 
+  force_sandbox_create: 'SFDX: Create a Sandbox Org...',
```

Comment:
I mentioned in another instance about do we need "Org" because we usually just say "sandbox" standalone